blocks/library/quote/index.js

```diff
@@ -0,0 +1,50 @@
+/**
+ * Internal dependencies
+ */
+import './style.scss';
+import { registerBlock, query as hpq } from 'api';
+import Editable from 'components/editable';
+
+const { html, query } = hpq;
+
+registerBlock( 'core/quote', {
+	title: wp.i18n.__( 'Quote' ),
+	icon: 'format-quote',
+	category: 'common',
+
+	attributes: {
+		value: ( node ) => query( 'blockquote > p', html() )( node ).join(),
+		citation: html( 'footer' )
+	},
+
+	edit( { attributes, setAttributes } ) {
+		const { value, citation } = attributes;
+
+		return (
+			<blockquote className="blocks-quote">
+				<Editable
+					value={ value }
+					onChange={ ( newValue ) => setAttributes( { value: newValue } ) } />
+				<footer>
```

Comment:
I wanted to hide this when the citation is empty and we're not focusing the block but it's not possible for now, because the `isSelected` is not enough for this. We need a distinct `isFocused` which stays active even if we start typing.